apps/web/components/pages/task/details-section/blocks/task-progress.tsx

```diff
@@ -135,7 +135,7 @@ const TaskProgress = () => {
 			<TaskRow labelTitle={trans.TOTAL_GROUP_TIME}>
 				<Disclosure>
 					{({ open }) => (
-						<div className="flex flex-col w-full">
+						<div className="flex flex-col w-full mt-[0.1875rem]">
```

Comment:
<img width="436" alt="image" src="https://github.com/ever-co/ever-teams/assets/81486442/d8973d75-8521-47c8-8244-39e4ec041cd3">

this is also not centered properly